### DISCUSSION TITLE
Allow to specify ADMIN_TOKEN in ENV

### DISCUSSION THREAD

**Discussion by tarmolehtpuu** [directus/directus#22059] [Feature Requests]
### Summary

Kubernetes initial ADMIN_EMAIL and ADMIN_PASSWORD can be specified via ENV, it would be good if ADMIN_TOKEN could be specified also

### Basic Example

A basic POC PR would be something like the following:

![Screenshot 2024-06-10 at 15 09 07](https://github.com/directus/directus/assets/115003/cf25189f-cfc3-43d5-8633-6bbbeabea55c)

### Motivation

We have Directus running in multiple environments and we don't want to tweak Directus manually, it would be good if all settings could be specified via ENV in code

### Detailed Design

I am not too familiar with the schema, but probably should be able to set the value of the token on the initially created user

### Must Have:
- Allow to specifiy ADMIN_TOKEN via ENV

### Drawbacks

N/A

### Alternatives

N/A

### Adoption Strategy

Not a breaking change, people who do not need this feature can ignore the ENV variable

**Comment by burka** [comment]
Added a pull request for this feature: https://github.com/directus/directus/pull/22724

**Comment by rijkvanzanten** [comment]
This was implemented in https://github.com/directus/directus/pull/22724!

**Reply by burka** [reply]
Awesome, thank you!
